moped-editor/src/views/projects/projectView/ProjectTimeline.js

```diff
@@ -434,6 +434,11 @@ const ProjectTimeline = ({ refetch: refetchSummary }) => {
     {
       title: "Related phase",
       field: "moped_milestone",
+      editable: "never",
+      cellStyle: {
+        fontFamily: typography.fontFamily,
```

Comment:
Not entirely sure this font family is needed here because I think this cell inherits it from the row styles but I could be wrong. Not a big deal at all.